### ISSUE TITLE
Updating shackle functionality in Achaea

### ISSUE THREAD

**Issue by antar5** [IRE-Mudlet-Mapping/ire-mapping-script#113]
Probably a relic from when mass used to do damage on movement, but shackle use by mapper can be made more efficient.

Changes that can be made:
- No longer remove shackle on normal speedwalking when the config option for shackle is enabled. Mass has no impact on normal speedwalking, so its unnecessary.

-With shackle option enabled, remove shackle only when the dash argument and its sister equivalents are included. Can do this two ways - easier to just remove shackle at the start of speedwalking and wear it once completed, like it currently does. Or can remove the shackle before each individual dash and wear it after the dash is completed, which is all that is necessary.

**Comment by QwindorRousseau** [comment]
I had decided when I made the function and the option was just to have it unwear at start and wear at end just to keep down on the spam and commands being sent.

**Comment by antar5** [comment]
Yeah, I figured that'd be the case. That only eliminates removing shackle on individual dashes, it should still be changed to only remove shackle if the dash or equivalent arguments are including in speedwalking. There are no downsides to this as far as I know (beneficial, really, since you ideally don't want shackle off for no reason).

**Comment by QwindorRousseau** [comment]
Did they ninja change that running around while massed didn't start to hurt your health after moving for a ton of rooms then? I know i still get the out of breath message when i follow someone who does that and I don't remove shackle but I haven't paid attention to the health amount.

**Comment by antar5** [comment]
Yeah, you no longer take damage from moving while massed.
